app/Mixins/MainView.js

```diff
@@ -0,0 +1,46 @@
+import Notification from '../Services/Notification';
+
+export function hasEntityAndView(entityName) {
+    try {
+        const view = this.getView(entityName);
+
+        if (!view.enabled) {
+            console.error(`${this.viewName} is disable for "${entityName}" entity.`);
+
+            return false;
+        }
+
+        return true;
+    } catch (e) {
+        if (console) {
+            console.error(e);
+        }
+
+        return false;
+    }
+}
+
+export function getView(entityName) {
```

Comment:
Not sure but I think in ES6, you can set default value:

```js
export function getView(entityName = null) {
```

This would make it easier to guess was is required or not.